JS-Algorithms/challenges.js

```diff
@@ -95,7 +120,20 @@ Since 10! === 3628800 and you sum 3 + 6 + 2 + 8 + 8 + 0 + 0
 ***** */
 
 const digitSum = (n) => {
-  // YOUR CODE HERE...
+  let factorial = 1n;
+  for (let i = 1n; i <= BigInt(n); i++) {
+    factorial = factorial * i;
+  }
+  
+  let digits = factorial.toString().split('');
+  let sum = 0;
+  for(let i = 0; i < digits.length; i++){
+    let digit = Number(digits[i]);
```

Comment:
Should we remove this extra variable?
```suggestion
    sum += Number(digits[i]);
```